docs/src/model_setup/buoyancy_and_equation_of_state.md

```diff
@@ -20,21 +21,22 @@ end
 ```jldoctest buoyancy
 julia> grid = RectilinearGrid(size=(64, 64, 64), extent=(1, 1, 1));
 
-julia> model = NonhydrostaticModel(grid=grid, buoyancy=nothing)
+julia> model = NonhydrostaticModel(grid=grid)
 NonhydrostaticModel{CPU, Float64}(time = 0 seconds, iteration = 0)
 ├── grid: RectilinearGrid{Float64, Periodic, Periodic, Bounded}(Nx=64, Ny=64, Nz=64)
-├── tracers: (:T, :S)
+├── tracers: ()
 ├── closure: Nothing
 ├── buoyancy: Nothing
 └── coriolis: Nothing
 ```
 
-In this case, you might want to explicitly specify which tracers to evolve. In particular, you may
-not want to evolve temperature and salinity, which are included by default. To specify no tracers,
-also pass`tracers = ()` to the model constructor.
+
+Alternatively, if you want to be explicit, you can simply pass `buoyancy = nothing` to the model constructor.
 
 ```jldoctest buoyancy
-julia> model = NonhydrostaticModel(grid=grid, buoyancy=nothing, tracers=())
+julia> grid = RectilinearGrid(size=(64, 64, 64), extent=(1, 1, 1));
```

Comment:
why did you have to add the grid here? how come the test was passing before?

Reply:
I guess I didn't have to. It just makes this little snippet more self-contained. It passes withouot it because grid is defined in the previous snippet. I can remove it if you prefer.